x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management/saved_objects_mix.json

```diff
@@ -0,0 +1,3091 @@
+{
```

Comment:
nit: this file is actually `ndjson` so changing the file extension would make editors stop complaining that it's not valid `json`

Reply:
I agree, but looks like all the kbnArchives in Kibana repo has `.json` extension and it is probably on purpose:
<img width="776" alt="image" src="https://user-images.githubusercontent.com/10977896/208910392-70c83d89-6c10-4a01-bfea-4078b311b323.png">

When I try to change it to  `ndjson`, I'm getting the following error:

```
Error: unable to resolve path [x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management/saved_objects_mix] to import/export, resolved relative to [/Users/dmle/github/kibana]
 proc [functional-tests]      │       at KbnClientImportExport.resolveAndValidatePath (node_modules/@kbn/test/target_node/src/kbn_client/kbn_client_import_export.js:42:13)
 proc [functional-tests]      │       at KbnClientImportExport.load (node_modules/@kbn/test/target_node/src/kbn_client/kbn_client_import_export.js:47:22)
 proc [functional-tests]      │       at /Users/dmle/github/kibana/node_modules/@kbn/journeys/target_node/journey/journey_ftr_harness.js:136:44
```

Reply:
Ah I see this is a bug in kbnArchiver https://github.com/elastic/kibana/blob/main/packages/kbn-test/src/kbn_client/kbn_client_import_export.ts#L38